Packs/FeedAWS/Integrations/FeedAWS/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Renamed "sub-feed" to "service".
```

Comment:
I am pretty sure that when there is only one release note for a release section, we don't add the `-`.